### DISCUSSION TITLE
Inquiry about Bandwidth Reset in Hobby Plan

### DISCUSSION THREAD

**Discussion by longtkhd** [vercel/community#2997] [Help] [answered]
### Summary

Bandwidth of Hobby plan

### Steps to Reproduce

Dear Vercel Support Team,
I am reaching out to inquire about resetting the bandwidth in my account. Currently, I am subscribed to the Hobby plan and have reached the bandwidth limit.

I would like to know if there is a way to reset the bandwidth to zero every month within the Hobby plan. If so, could you please provide me with information on the process and any conditions associated with this reset? Additionally, I would appreciate further clarification on the fees and regulations applied when exceeding the bandwidth limit in the Hobby plan. Kindly inform me about any overage fees and the specific terms and conditions related to this situation.

Thank you very much for your assistance.
I look forward to your prompt response.

**Comment by amyegan** [comment, marked as answer]
Hi @longtkhd. You can find more information about usage limits here: https://vercel.com/docs/concepts/limits/overview

You can find links to pricing and limits for individual features from that page.
